download_and_install/compiling_from_source.md

```diff
@@ -0,0 +1,242 @@
+Compiling From Source
```

Comment:
Is it worth a bit of an introduction here pointing people at the package instructions where possible (in case they hit these instructions via search)?

It strikes me they probably only want to compile from source if:
- It's not available on their OS
- They want to fix something/test a fix
- They want to try the bleeding edge.